packages/react-atlas-core/webpack.config.js

```diff
@@ -11,6 +11,21 @@ module.exports = {
     publicPath: '/lib/',
     libraryTarget: 'commonjs2'
   },
+  externals: [{
```

Comment:
How come you have yo use externals for react and react dom, why can't you use the react and react dom that comes with react-atlas?